9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
I am not sure if I understand. why we are talking about mysql? this is postgresql container. Or you mean example queries from myslq container? AFAIK queries should be same/similar since it's SQL.